src/components/Player.vue

```diff
@@ -219,10 +222,12 @@ export default {
                                 (component.getPreferenceBoolean("proxyLBRY", false) || headers.Range))
                         ) {
                             url.searchParams.set("host", url.host);
-                            url.host = proxyHost;
+                            url.port = proxyURL.port;
+                            url.host = proxyURL.host;
+                            url.pathname = proxyURL.pathname + url.pathname;
                             request.uris[0] = url.toString();
                         }
-                        if (url.pathname === "/videoplayback") {
+                        if (url.pathname === proxyURL.pathname + "/videoplayback") {
```

Comment:
Same problem as above.

Reply:
I think this just has to be `proxyPath`, and then it looks good.

Reply:
fixed it